apps/experiments/views/chat.py

```diff
@@ -0,0 +1,21 @@
+from django.http import Http404
+from django.shortcuts import get_object_or_404, render
+
+from apps.chat.models import ChatMessage, ChatMessageType
+
+
+def rate_message(request, team_slug: str, message_id: int, rating: str):
+    if rating not in ["👍", "👎"]:
```

Comment:
will we have any issues with these emojis being updated/modified?

Reply:
wondering if we should use the html encodings or not `&#128077;` `&#128078;`

Reply:
These emoji's are part of unicode, which should make it fine to include as-is? Or am I missing something?

Reply:
The main concern if are we sure that all of our users are on environments that support Unicode emojis? I know there are some older software so I wonder if we should error on the safe side

Reply:
we can circle back on this in a future convo